### PR TITLE
Support Renaming Containers 

### PR DESCRIPTION
**Pull request by RosalineP**
#### Rationale
We are seeking to add the ability to rename a container to our APIs.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/4335

#### Changes
* Remove outdated section of readme
* Add new command and response
* Update changelog
* Bump version as specified in readme

### CODE REVIEW

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Something strange is going on here. It looks like this FB was created from an old branch or tag, not develop. For example, CHANGELOG.md is not showing the 5.1.0 changes (that's the latest release and what's on develop). I think you need to rebase this FB to develop.

File: src/org/labkey/remoteapi/security/RenameContainerCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ */
+public class RenameContainerCommand extends PostCommand<RenameContainerResponse> {
+    private String _name;
+    private String _title;
+    private boolean _addAlias = true;
+
+
+    public RenameContainerCommand(String name, String title, boolean addAlias)
+    {
+        super("core", "RenameContainer");
```
Comment (Sigmonia) [inline src/org/labkey/remoteapi/security/RenameContainerCommand.java]: This is the wrong controller
```suggestion
        super("admin", "RenameContainer");
```

**Review by Sigmonia — CHANGES_REQUESTED** [review]

File: src/org/labkey/remoteapi/security/RenameContainerCommand.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        return result;
+    }
+
+    public String get_name() {
+        return _name;
+    }
+
+    public void set_name(String _name) {
+        this._name = _name;
+    }
```
Comment (labkey-adam) [inline src/org/labkey/remoteapi/security/RenameContainerCommand.java:59]: IntelliJ may have generated these getter and setter names by default, but our usual pattern for all getters and setters is `getName()` and `setName(String name)`. Then `_name = name` (no need for `this._name`).

**Review by labkey-adam — CHANGES_REQUESTED** [review]

File: src/org/labkey/remoteapi/security/RenameContainerCommand.java (hunk 2 above)
Comment (RosalineP) [reply src/org/labkey/remoteapi/security/RenameContainerCommand.java:59]: Indeed these were intelliJ generated—lots of weirdness going on, regenerating the getters + setters reverted to our usual pattern. Thanks for the catches Adam!

**Comment by RosalineP** [comment]
Considering adding more constructors, since only one of 'name' and 'title' is ever required, and 'addAlias' defaults true on the server action

File: src/org/labkey/remoteapi/security/RenameContainerCommand.java (hunk 2 above)
Comment (Sigmonia) [reply src/org/labkey/remoteapi/security/RenameContainerCommand.java:59]: I think a builder pattern to set the values similar to some of the other commands would work well.

Comment (RosalineP) [reply src/org/labkey/remoteapi/security/RenameContainerCommand.java:59]: Ian and I talked about this offline and decided to leave it as-is currently

**Review by Sigmonia — APPROVED** [review]

**Review by labkey-adam — APPROVED** [review]

**Pull request merged**